src/services/queue/sqs-queue-service.ts

```diff
@@ -123,6 +125,13 @@ export class SqsQueueService<TValue extends QueueMessageData>
       region: config.queue.awsRegion,
       endpoint: this.sqsQueueUrl,
       logger: awsLogger,
+      requestHandler: new NodeHttpHandler({
+        httpAgent: new http.Agent({
+          keepAlive: true,
+          maxSockets: 50,
+          maxFreeSockets: 20,
```

Comment:
Why set this one? Are we worried about the default? https://nodejs.org/dist/latest-v4.x/docs/api/http.html#http_agent_maxfreesockets

Reply:
256 seems a lot, added this to mitigate the extra overhead, if max is 50

Reply:
Yeah, it would basically mean all 50. 256 is a lot less than the other default of infinity (infinitely less even lol). I don’t really mind either way but I’m not sure it’s worth modifying unless we have a symptom like running out of file ulimits or seeing memory pressure. Setting it makes it seem like we expect it to matter 🤷

Reply:
If we aren’t sure what we want yet and want to experiment a bit, we could have `process.env.someGoodVarName || 50`. Not sure it’s worth exposing as config, but it’s an option

Reply:
Also (sorry to pile on with comments), the title isn’t quite right as 50 should be the default so we’re not increasing it, and 20 free is less than the default. It’s the reuse that’s being modified

Reply:
That's problematic, let me bump these up. I think 256 is not enough, let me double it then to 512

Reply:
removing these, let's tart with just the keep alive. Reuse the connections that should reduce the number of dns lookup calls